neard/src/cli.rs

```diff
@@ -493,12 +493,29 @@ pub(super) struct RecompressStorageSubCommand {
     /// Directory where to save new storage.
     #[clap(long)]
     output_dir: PathBuf,
+
+    /// Keep data in ColPartialChunks column.  This is always true if node is
+    /// not an archival node.
+    #[clap(long)]
+    keep_partial_chunks: bool,
+
+    /// Keep data in ColTrieChanges column.  This is always true if node is
+    /// not an archival node.
+    #[clap(long)]
+    keep_trie_changes: bool,
 }
 
 impl RecompressStorageSubCommand {
     pub(super) fn run(self, home_dir: &Path) {
-        if let Err(err) = nearcore::recompress_storage(&home_dir, &self.output_dir) {
+        warn!(target: "neard", "Recompressing storage; note that this operation may take up to a day to finish.");
```

Comment:
and if it is disturbed, you have to start from scratch, right?

Reply:
Yep.